src/main/java/com/minecolonies/commands/ColoniesCommand.java

```diff
@@ -0,0 +1,37 @@
+package com.minecolonies.commands;
+
+import com.google.common.collect.ImmutableMap;
+import net.minecraft.command.ICommand;
+import net.minecraft.command.ICommandSender;
+import net.minecraft.server.MinecraftServer;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Minecolonies root command.
+ * <p>
+ * Manages all sub commands.
+ */
+public class ColoniesCommand extends SplitCommand
+{
+
+    public ColoniesCommand()
+    {
+        super(new String[] {"mc", "colonies"});
+    }
+
+    private ImmutableMap<String, ISubCommand> subCommands =
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Private field 'subCommands' could be made final; it is only initialized in the declaration or constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3AImmutableField)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1213)